examples/apps/ai-collab/package.json

```diff
@@ -29,7 +29,30 @@
 		"start": "next dev",
 		"start:server": "tinylicious"
 	},
-	"dependencies": {},
+	"c8": {
+		"all": true,
+		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.*ts",
+			"dist/test/**/*.*js"
+		],
+		"exclude-after-remap": false,
+		"include": [
+			"src/**/*.*ts",
+			"dist/**/*.*js"
+		],
+		"report-dir": "nyc/report",
+		"reporter": [
+			"cobertura",
+			"html",
+			"text"
+		],
+		"temp-directory": "nyc/.nyc_output"
+	},
```

Comment:
We don't really need code coverage for the example apps (we don't even have tests in this one :) ).
```suggestion
```